.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: '12.x'
-      - run: npm install && npm --prefix packages/icons install
+      - run: npm install && npm --prefix packages/icons install && yarn --cwd packages/angular
```

Comment:
Yarn gets removed in the commit after `get angular building and testing` Correct?

Reply:
Yes, I missed this reference before my first push.